LayoutTests/accessibility/mac/invalid-tree-root-at-iframe.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html aria-label="main-page-web-area">
+<head>
+<script src="../../resources/accessibility-helper.js"></script>
+<script src="../../resources/js-test.js"></script>
+<script src="../../resources/ui-helper.js"></script>
+</head>
+<body>
+
+<iframe id="iframe" onload="runTest()" aria-hidden="true" srcdoc="<html aria-label='iframe-web-area'><button>Inside iframe</button></html>"></iframe>
```

Comment:
I think you should be able to put newlines in srcdoc, which might make this more readable. (Unless there's a reason that breaks the test.)